backend/routes/content.py

```
@@ -391,7 +437,98 @@ def mark_module_complete_and_open_next(module_id, user_id):
+
+        logger.info(f"Solution module {solution_module.id} opened for user")
+        return (
+            jsonify(
+                {
+                    "solutionId": solution_module.id,
+                    "moduleType": solution_module.module_type.name,
+                }
+            ),
+            200,
+        )
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)